src/data_gradients/feature_extractors/common/image_duplicates.py

```diff
@@ -0,0 +1,264 @@
+from collections import Iterable
+from typing import List
+from data_gradients.common.registry.registry import register_feature_extractor
+from data_gradients.feature_extractors.abstract_feature_extractor import AbstractFeatureExtractor
+from data_gradients.feature_extractors.abstract_feature_extractor import Feature
+from imagededup.methods import DHash
```

Comment:
We should add this `imagededup` to the `requirements.txt`.
But I'm not sure if it us good to force users to install it or not. Does this lib require many other libs ? If yes, we may want to add it as an optional requirement.

Reply:
it is in requirements.txt.
- Not such a big deal IMO https://github.com/Deci-AI/data-gradients/blob/master/requirements.txt
- I would argue that this is a pretty important feature, so IMO we should keep it mandatory.